src/types.ts

```diff
@@ -14,9 +14,7 @@
 	limitations under the License.
 */
 
-import { Dimension } from "./utils/metric/metric";
-// re-export type because it is used in the config
-export { Dimension } from "./utils/metric/metric";
```

Comment:
Why did you remove this export?

Reply:
As I understand it, we imported that and exported it in order to use it in code that consumes this exporter right? But it doesn't show up in autocomplete either way (even if I add it back), and the example seems to be working, so I wasn't sure if it was needed. But I am at the very beginning of my understanding of npm packages, so I am not sure if its needed.

Reply:
Hum yeah if the users don't also install our utils package, then they will not get this type? I tried by creating a new node app, installing just this dependency and then tried to used it as: 

![image](https://user-images.githubusercontent.com/5938087/136060616-3ee77ebc-9169-458a-8273-381a05fa581e.png)

There is some auto-complete when I try to create the ExporterConfig, but of course the `Dimension` type is not there.